client/src/components/settings/About.js

```diff
@@ -105,6 +111,12 @@ class AboutSettings extends Component {
 
   handlePictureChange = e => {
     const value = e.target.value.slice(0);
+    var pattern = /^((https?|ftp):)?\/\/.*(jpeg|jpg|png|gif|bmp)$/;
```

Comment:
An image URL may not have an extension at all times.

Reply:
Additionally, I don't think we should use `var` here.

Reply:
can you check now ?